src/store/user/reducer.js

```diff
@@ -0,0 +1,27 @@
+import { LOGIN_USER_INFO, LOGOUT_USER_INFO } from './types';
+
+const userInitialState = {
+	isAuth: false,
+	name: '',
+	email: '',
+	token: '',
+};
+
+export default function userReducer(state = userInitialState, action) {
+	switch (action.type) {
+		case LOGIN_USER_INFO:
+			return {
+				...state,
+				...action.payload,
+			};
+
+		case LOGOUT_USER_INFO:
+			return {
+				...state,
```

Comment:
It seems you can use there just userInitialState, without state + userInitialState

Reply:
fixed, thanks